### PR TITLE
Suggested Security Policy

### PR DESCRIPTION
**Pull request by kpolchow**
This is a suggested Security Policy file for GRASS, and will appear when users click on the Security tab.

### CODE REVIEW

File: SECURITY.md
Comment (echoix) [inline SECURITY.md:5]: Do we want to offer the GitHub way to privately report a security vulnerability too as an alternative? If it enabled here? It's hard for people with already write access to see it, and I don't have a second account to test it.

https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability

**Review by echoix — APPROVED** [review]

**Review by kpolchow — COMMENTED** [review]
Made a commit to add link to security tab

**Review by echoix — DISMISSED** [review]
I think this is correct and appropriate. However, it's a special kind of PR, and I would've liked a little bit more feedback from others in the project. I understand that it was created with @wenzeslaus in the background.

I would be ready to merge it right now (and even since you applied the last suggestions).

Why I think it needs more than my simple approval: it formalizes our processes and changes a bit the responsibilities in the project, effectively creating a new role, if Vaclav Petras is the only one receiving special security reports at that email address. (Hoping we never need to handle this) Some long discussions have been made on RFCs for less than that. 
Through the GitHub interface, that responsibility would be shared along many members, as more than one would see the issue separately, and probably be notified by some sort.
I still think that having a separate, independent communication channel, like with an university email like this, that can be traced back to a physical presence if needed, is required. It would be the reporter's choice to choose what communication channel is the best for the type of vulnerability in play. (Ie: a compromise of one of the member's GitHub account, or a problem with the GitHub itself, would be beneficial to not have to report through that comprised platform)

So, can some of others jump in for deciding on what to do next? Would this be merged before the end of the discussion? The text itself and the motivation isn't the problem here, it's only about the process.

File: SECURITY.md
Comment (wenzeslaus) [inline SECURITY.md:5]: There is no need to have anything besides standard GitHub reporting:

```suggestion
1. **Privately disclose the issue** by submitting a Security Advisory through [GitHub Security](https://github.com/OSGeo/grass/security). Please do not create publicly viewable issues for security vulnerabilities.
```

File: SECURITY.md
Comment (wenzeslaus) [inline SECURITY.md:27]: This should link our release procedure or release guidelines. I think the best document at this point is [Trac wiki page Release Schedule](https://trac.osgeo.org/grass/wiki/Release/Schedule). Replace the table by a sentence referring people to that page. Something along the lines of "Refer to [Release Schedule](https://trac.osgeo.org/grass/wiki/Release/Schedule) for details about which versions are supported."

**Review by wenzeslaus — CHANGES_REQUESTED** [review]

**Review by wenzeslaus — CHANGES_REQUESTED** [review]

File: SECURITY.md
Comment (wenzeslaus) [inline SECURITY.md:33]:
```suggestion
While packages in Linux and other unix-like distributions are out of scope of this document, distribution maintainers traditionally do a great job in patching their distributions for security vulnerabilities. Please, refer to a specific distribution or package source if you are using packages for a specific software distribution.
```

**Review by wenzeslaus — APPROVED** [review]
Thanks you! While I assume we may be doing changes in the future, this is a great document for now.

I think we can use this as us and we can incorporate feedback in separate PRs. Alternatively, we can wait with this one open for couple days or weeks for feedback here.

**Comment by echoix** [comment]
I think that we could merge it as of now, as there has already been almost a month to manifest itself, and we will have the chance to talk about it at the community meeting or before. Now that the email assigning one person is removed, I don't see any big contentious points, what remains is the implementation of the procedures themselves, that is out of the scope of the PR, @kpolchow has finished his job.

**Review by echoix — APPROVED** [review]

**Comment by wenzeslaus** [comment]
Merged. Thank you @kpolchow!

**Pull request merged**
